.github/CODEOWNERS

```diff
@@ -70,6 +70,7 @@
 /core/domain/opportunity*.py @sagangwee
 /core/domain/voiceover_services*.py @sagangwee
 /core/storage/opportunity/ @sagangwee
+/core/templates/domain/community_dashboard/ @sagangwee
 /core/templates/domain/opportunity/ @sagangwee
```

Comment:
Should we move the new domain object file to this opportunity as opportunity sounds like a domain class and the new class is also related to opportunity. Any thoughts?

Reply:
Done